test/mocks/objects/notify-link-to-file.js

```diff
@@ -10,7 +10,20 @@ const confirmEmail = {
   confirm_email_before_download: true
 }
 
+const retentionPeriodSet = {
+  ...defaultObject,
+  retention_period: '78 weeks'
+}
+
+const confirmEmailAndRetentionPeriodSet = {
+  ...defaultObject,
+  confirm_email_before_download: true,
+  retention_period: '78 weeks'
+}
```

Comment:
And here from line 15

Reply:
That was the situation except the the name before your comment below; It has been reverted to the initial situation with new combined object name.

https://github.com/DEFRA/ffc-pay-statement-publisher/pull/14#discussion_r1151712742

Reply:
But with those defined `confirmEmail` and `retentionPeriodSet` we can combine them into the 1 joined object using their objects themselves as give in this code snippet;

```
const confirmEmailAndRetentionPeriodSet = { 
    ...defaultObject, 
    ...confirmEmail, 
    ...retentionPeriodSet 
}
```

Reply:
Acc'ly don't need `defaultObject` here can just use the 2 objects